### PR TITLE
Cypress/E2E: Fix react_to_center and close_current_dm_redirect specs

### PR DESCRIPTION
**Pull request by josephbaylon**
#### Summary
- Fixed react_to_center_spec
- Fixed closed_current_dm_redirect spec
- Added `KNOWN ISSUE` to title of failing test cases with known issues that are currently unresolved

#### Ticket Link
NA

#### Screenshots
Screen Shot 2022-01-31 at 10 09 32 AM
Screen Shot 2022-01-31 at 10 32 04 AM

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: e2e/cypress/integration/files_and_attachments/image_link_preview_spec.js [hunk 1]
```diff
@@ -96,7 +96,7 @@ describe('Image Link Preview', () => {
         cy.findAllByLabelText('file thumbnail').should('be.visible').and('have.length', 4);
     });
 
-    it('MM-T332 Image link preview - Bitly links for images and YouTube', () => {
+    it('MM-T332 Image link preview - Bitly links for images and YouTube -- KNOWN ISSUE: MM-40448', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/files_and_attachments/image_link_preview_spec.js:99]: I'm fine with this approach as we initially discussed offline. Please document this new process at https://developers.mattermost.com/contribute/webapp/e2e/guide-for-writing/

**Review by saturninoabril — APPROVED** [review]
Thanks @josephbaylon!

File: e2e/cypress/integration/files_and_attachments/image_link_preview_spec.js (hunk 1 above)
Comment (josephbaylon) [reply e2e/cypress/integration/files_and_attachments/image_link_preview_spec.js:99]: @saturninoabril Created a doc PR https://github.com/mattermost/mattermost-developer-documentation/pull/1005

Comment (saturninoabril) [reply e2e/cypress/integration/files_and_attachments/image_link_preview_spec.js:99]: Thanks!

**Review by furqanmlk — APPROVED** [review]
Thanks @josephbaylon LGTM.
I like the idea of having bug number in test cases.

**Pull request merged**